azure-pipelines.yml

```diff
@@ -234,8 +234,17 @@ stages:
           inputs:
             versionSpec: '$(python.version)'
           displayName: 'Use Python $(python.version)'
+        - bash: echo "##vso[task.prependpath]$CONDA/bin"
+          displayName: Add conda to PATH
```

Comment:
This looks like it just echos the conda path. Maybe it doesn't need to be added to PATH?

Reply:
I just copied this from: https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/anaconda?view=azure-devops&tabs=macos